rental.rb

```diff
@@ -0,0 +1,30 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    update_book(book)
+    update_person(person)
+  end
+
+  def person=(person)
+    update_person(person)
+  end
+
+  def book=(book)
+    update_book(book)
+  end
+
+  private
+
+  def update_book(book)
+    book.rentals.push(self) unless book.rentals.include?(self)
+    @book = book
+  end

```

Comment:
- [ ] As you can see you are accessing the rentals here. Please add setters for it in the Book class.